src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,228 @@
+package src.main.java.duke.parser;
+
+import static src.main.java.duke.commons.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static src.main.java.duke.commons.Messages.MESSAGE_INVALID_TASK_DISPLAYED_INDEX;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import src.main.java.duke.commands.AddDeadlineCommand;
+import src.main.java.duke.commands.AddEventCommand;
+import src.main.java.duke.commands.AddTodoCommand;
+import src.main.java.duke.commands.Command;
+import src.main.java.duke.commands.DeleteCommand;
+import src.main.java.duke.commands.ExitCommand;
+import src.main.java.duke.commands.FindCommand;
+import src.main.java.duke.commands.HelpCommand;
+import src.main.java.duke.commands.IncorrectCommand;
+import src.main.java.duke.commands.ListCommand;
+import src.main.java.duke.commands.MarkDoneCommand;
```

Comment:
I like how you have organized your import statements. Very neat and easy to read :)

Reply:
Thank you :)